python/src/Sample_doc.i.in

```diff
@@ -81,6 +81,28 @@ experiments
 1 : [ 2 3 ]
 2 : [ 4 5 ]"
 
+
+%feature("docstring") OT::Sample::BuildFromSequence
+"Static method for building a sample from a sequence of float.
+
+Parameters
+----------
+data : 1d array-like
+    Data.
+
+Returns
+-------
+sample : :class:`~openturns.Sample`
+    Sample generated from sequence
+
+Examples
+--------
+>>> import openturns as ot
+>>> n = 20
+>>> x = ot.Sample_BuildFromSequence(range(n))
```

Comment:
x,n are useless here

Reply:
hmm why? we just want to build a sample from a list `[0,1,...,n-1]`

Reply:
oh yes, nevermind